src/main/java/net/sf/mzmine/modules/peaklistmethods/io/gnpsexport/GNPSExportTask.java

```diff
@@ -53,7 +53,7 @@
 
   @Override
   public double getFinishedPercentage() {
-    return 0;
+    return 1;
```

Comment:
Could you add a proper progress estimate calculated from the number of peak list rows exported? 
See other modules for inspiration (it is a trivial calculation)